talisker/testing.py

```diff
@@ -283,11 +283,11 @@ def __init__(self, name=None):
         self.sentry_context = talisker.sentry.TestSentryContext(self.dsn)
 
     def start(self):
-        Context.new()
         self.old_statsd = talisker.statsd.get_client.raw_update(
             self.statsd_client)
         talisker.logs.add_talisker_handler(logging.NOTSET, self.handler)
         self.sentry_context.start()
+        Context.new()
 
```

Comment:
FTR - this is a driveby UX improvement - the previous lines generates logs, moving the new context to here means those logs are not captured as part of the context.